apps/service/src/app/(blank)/onBoarding/_components/shorts/ShortsExperienceTab.tsx

```
@@ -274,8 +272,12 @@ export const ShortsExperienceTab = React.memo(
       const sortedCategories = Array.from(categoryScores.entries()).sort(
         (a, b) => b[1] - a[1],
       );
-      // TODO : 이후 카테고리 정해지면, 음수 점수는 제외하고 보내기
-      return sortedCategories.slice(0, 5).map((category) => category[0]);
+      return sortedCategories
+        .slice(0, 5)
+        .map((category) => {
```

Biome (2.4.7)

[error] This callback passed to map() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)